outdated/modules/httpd.reb

```diff
@@ -225,199 +256,221 @@ sys/make-scheme [
+                header-feed content: to end (
+                    binary: copy :content
+                    content: does [content: as-text binary]
+                )
+            ] else [
+                comment [ ;-- !!! was commented out, why?
+                    action: _
+                    target: _
+                    request-method: _
+                    query-string: _
+                    binary: _
+                    content: _
+                    request-uri: _
+                ]
+                net-utils/net-log error: "Could Not Parse Request"
+                return _
+            ]
 
-                    length: content-length: any [
-                        attempt [length: to integer! length]
-                        0
-                    ]
+            version: to text! :version
+            request-method: method: to text! :method
+            path-info: target: as-text :target
+            action: spaced [method target]
+            request-uri: as-text request-uri
+            server-port: query/mode client 'local-port
+            remote-addr: query/mode client 'remote-ip
 
-                    net-utils/net-log action
-                ][
-                    ; action: target: request-method: query-string: binary: content: request-uri: _
-                    net-utils/net-log error: "Could Not Parse Request"
-                ]
+            headers: make header-prototype
+                <- http-headers: new-line/skip headers true 2
+
+            type: try all [
+                text? type: headers/Content-Type
+                copy/part type (opt find type ";")
             ]
-        ]
-    ]
 
-    dispatch: use [status-codes build-header hdr][
-        status-codes: [
-            200 "OK" 201 "Created" 204 "No Content"
-            301 "Moved Permanently" 302 "Moved temporarily" 303 "See Other" 307 "Temporary Redirect"
-            400 "Bad Request" 401 "No Authorization" 403 "Forbidden" 404 "Not Found" 411 "Length Required"
-            500 "Internal Server Error" 503 "Service Unavailable"
+            length: content-length: attempt [to integer! length] else [0]
+
+            net-utils/net-log action
         ]
+    ]
 
-        build-header: func [response [object!]][
+    dispatch: function [
+        client [port!]
+
+      <static>
+
+        status-codes ([
+            200 "OK"
+            201 "Created"
+            204 "No Content"
+            
+            301 "Moved Permanently"
+            302 "Moved temporarily"
+            303 "See Other"
+            307 "Temporary Redirect"
+            
+            400 "Bad Request"
+            401 "No Authorization"
+            403 "Forbidden"
+            404 "Not Found"
+            411 "Length Required"
+
+            500 "Internal Server Error"
+            503 "Service Unavailable"
+        ])
+
+        build-header (function [response [object!]] [
             append make binary! 1024 spaced collect [
-                case/all [
```

Comment:
Stylistic change: use of CASE/ALL without a THEN or ELSE or something doesn't make much sense.  There's no shortcut, so there's not much value.  It forces a level of indentation just to avoid putting an IF on each line...so really just makes code harder to understand and change from the structure.